src/spectral/utils/spectral-validator.js

```diff
@@ -57,12 +57,19 @@ const setup = async function(spectral) {
   spectral.registerFormat('oas3', isOpenApiv3);
 
   // load the spectral ruleset, either a user's or the default ruleset
-  const spectralRuleset = await config.getSpectralRuleset(
-    defaultSpectralRuleset
+  const spectralRulesetURI = await config.getSpectralRuleset(
+    defaultSpectralRulesetURI
   );
 
+  // Combine user ruleset with the default ruleset
+  // The default ruleset will take precendence in any rules it defines (including extended rules)
+  await spectral.loadRuleset([spectralRulesetURI, defaultSpectralRulesetURI]);
```

Comment:
I dont think there is a problem with loading a ruleset twice since it won't create any duplicates. Currently it will load the user config and the `.defaultsForSpectral.yaml` config or the `.defaultsForSpectral.yaml` config twice, how should this be changed? I can change it to load the `.defaultsForSpectral.yaml` first, should I make any other changes?

Reply:
Okay. Yes, I think we should load the ``.defaultsForSpectral.yaml `` first

Reply:
Ok i updated it to load the defaults first